common/app/staticpages/StaticPages.scala

```diff
@@ -22,4 +22,14 @@ object StaticPages {
       contentType = GuardianContentTypes.NetworkFront,
       iosType = None,
       shouldGoogleIndex = false))
+
+    def simpleEmailSignupPage(id: String, webTitle: String): SimplePage = SimplePage(
+      MetaData.make(
+        id = id,
+        section = Option(SectionSummary(id="global", activeBrandings=None)),
+        webTitle = webTitle,
+        analyticsName = "global",
+        contentType = "survey",
```

Comment:
Yep, I need to still use the survey template until I can set up a separate 'Signup' one...